### PR TITLE
New SDK function: context.getLairURL()

### PR DESCRIPTION
**Pull request by rachellhea**
## Change description

Exposing the new environment variable programmatically.

## Checklist

- [x] Application changes have been tested and automated tests have been added, if applicable
- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: test/utils.test.js [hunk 1]
```diff
@@ -18,6 +18,23 @@ test('Build Workspace Endpoint URL', () => {
...
+test('Get Environment Variables', () => {
+    let env_vars = [
+        "WAYSCRIPT_EXECUTION_USER_TOKEN",
+        "WAYSCRIPT_EXECUTION_USER_REFRESH_TOKEN",
+        "WAYSCRIPT_EXECUTION_USER_APPLICATION_KEY",
+        "WAYSCRIPT_LAIR_URL",
+        "WS_PROCESS_ID"
+    ];
+    
+    env_vars.forEach(v => process.env[v] = "TEST_SETTING");
```
Comment (aroxby-wayscript) [inline test/utils.test.js]: I highly recommend setting each of these to different values to ensure that the correct env var is read.  I notice we do something similar in python and I don't feel great about it.

Comment (aroxby-wayscript) [inline test/utils.test.js]: Is there someway we can clean up `process.env` when we're done.  Or mock it entirely?  If you were to test this on WayScript you would be overwriting useful values.

**Review by aroxby-wayscript — COMMENTED** [review]
I like it but the tests could be better.

File: test/utils.test.js (hunk 1 above)
Comment (rachellhea) [reply test/utils.test.js]: Done.

Comment (rachellhea) [reply test/utils.test.js]: Done.

**Review by samkam — APPROVED** [review]

**Pull request merged**